skeleton/cli-bundler/aurelia_project/tasks__if_typescript/transpile.ts

```diff
@@ -5,13 +5,15 @@ import * as rename from 'gulp-rename';
 import * as ts from 'gulp-typescript';
 import * as project from '../aurelia.json';
 import * as fs from 'fs';
+import { CLIOptions, build, Configuration } from 'aurelia-cli';
 import * as through from 'through2';
-import {CLIOptions, build} from 'aurelia-cli';
+import * as gulpSourcemaps from 'gulp-sourcemaps';
```

Comment:
You don't need gulp-sourcemaps.
gulp v4 has integrated sourcemaps support in gulp.src and gulp.desc, you can have a look of gulp doc.